junit-platform-launcher/src/main/java/org/junit/platform/launcher/Launcher.java

```diff
@@ -81,8 +81,24 @@
 	 * collecting their results, and notify {@linkplain #registerTestExecutionListeners
 	 * registered listeners} about the progress and results of the execution.
 	 *
+	 * <p>The default implementation delegates to
+	 * {@link #execute(LauncherDiscoveryRequest, TestExecutionListener...)} with no
+	 * additional listeners.
+	 *
 	 * @param launcherDiscoveryRequest the launcher discovery request; never {@code null}
 	 */
-	void execute(LauncherDiscoveryRequest launcherDiscoveryRequest);
+	default void execute(LauncherDiscoveryRequest launcherDiscoveryRequest) {
```

Comment:
Adding `@Deprected` to the method breaks the build, because of `error: warnings found and -Werror specified`. Will temporarely modify all no-arg calls to pass an extra `new TEListener[0]` argument to make the compiler happy. Or do you know a better way around this?